packages/webapp/src/webview/ui/components/UIComponentsLib/Icons/Icons.tsx

```diff
@@ -654,6 +656,22 @@ export const initIcons = function (): void {
                     />
                 </svg>
             ),
+            [UiIcons.MessageSent]: (
```

Comment:
Hi @sanat-thukral, in course of ticket https://github.com/SAP/guided-answers-extension/issues/364, @unseen1980 made sure that all icons we need are available in `@sap-ux/ui-components`. 

PR: https://github.com/SAP/guided-answers-extension/pull/387 removes now the local copy of icons component from this repo. I think we need to do the same for these icons here.

Reply:
Thanks for pointing that out Klaus, makes sense.